projects/test-app/src/app/state/customer.state.ts

```diff
@@ -1,7 +1,14 @@
 import { buildState, IEntityState } from '@briebug/ngrx-auto-entity';
 import { Customer } from '../models/customer.model';
 
-export const { initialState, facade: CustomerFacadeBase } = buildState(Customer);
-export function customerReducer(state = initialState): IEntityState<Customer> {
+export const { initialState, facade: CustomerFacadeBase } = buildState(Customer, {
+  customProperty: 'hello'
```

Comment:
Similar to the previous comment, was this purely for testing (or, in this case, perhaps it's to show custom properties can be used?)

Reply:
Yup, purely for testing.